src/index.js

```diff
@@ -1,15 +1,15 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './firebase.js';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React from "react";
+import ReactDOM from "react-dom";
+// import './firebase.js';
```

Comment:
we're going to need this back in here later I believe

Reply:
Didn't realize I deleted it. Putting it back